doc/source/dev/howto-docs.rst

```diff
@@ -0,0 +1,141 @@
+developers.
+
+If you're reading this page, you probably want to help. This guide is meant to
+help you decide which kind of content you'll write, as well as give you some
+tips and instructions for submitting it to the official NumPy documentation
+(that is, the documentation that ships with NumPy and lives on the official
+project pages). Keep in mind that if you don't want to do this, writing a
+tutorial on your own blog, creating a youtube video or answering questions on
+social media or Stack Overflow are also great contributions!
+
+NumPy has a Documentation team. We have open meetings on Zoom every three weeks
+and invite everyone to join. Don't hesitate to reach out if you have questions
+or just need someone to guide you through your first steps - we're always happy
+to help. Meetings are usually announced on the `numpy-discussion mailing list
+<https://mail.python.org/mailman/listinfo/numpy-discussion>`__. Meeting minutes
+are taken `on hackmd.io <https://hackmd.io/oB_boakvRqKR-_2jRV-Qjg>`__ and stored
+in the `NumPy Archive repository <https://github.com/numpy/archive>`__.
+
+You can find larger planned and in-progress documentation improvement ideas `at
+our GitHub project <https://github.com/orgs/numpy/projects/2>`__.
+
+Current vision for the documentation: NEP 44
+--------------------------------------------
+
+Recently, the NumPy community approved a *NumPy Enhancement Proposal (NEP)*
+about documentation, `NEP 44 - Restructuring the NumPy Documentation
+<https://www.numpy.org/neps/nep-0044-restructuring-numpy-docs>`__.
+
+**Where is the documentation?**
+
+The main page for the :ref:`NumPy Documentation <numpy_docs_mainpage>` lists
+several categories. The documents mentioned there live in different places.
+
+- **Tutorials, How-Tos, Explanations:** These documents are stored in the NumPy
+  source code tree, which means that in order to add them to the official
+  documentation, you have to download the NumPy source code,
+  :ref:`build it <howto-build-docs>` and submit your changes via a
+  :ref:`GitHub pull request <devindex>`.
+
+- **API Reference:** These are mainly the result of rendering the NumPy code
+  `docstrings <https://www.python.org/dev/peps/pep-0257/>`__ into formatted
+  documents. They are automatically generated when NumPy is built from source.
+
+**Datasets**
+
+If you are writing a tutorial or how-to, we encourage you to use real images and
+data (provided they are appropriately licensed and available). This makes the
+material more engaging for readers, and choosing the right data can add
+pedagogical value to your content.
+
+Creating new content
+--------------------
+
+The documentation is written in restructuredText, which is the format required
+by Sphinx, the tool most Python projects use to automatically build and link the
+documentation within the project. You can read this
+`Sphinx Quick Reference Guide
+<https://docutils.sourceforge.io/docs/user/rst/quickref.html>`__ for more
```

Comment:
The link title (i.e. "Sphinx Quick Reference Guide") is a little misleading as it actually points to the rST quick reference. Also, this link certainly has all the information, but the docutils docs are so ugly... maybe consider linking to sphinx's rST primer (http://www.sphinx-doc.org/en/stable/usage/restructuredtext/basics.html) instead?

Reply:
I'll do both, if that's ok. I think it's better to have more sources of information when possible.